src/main/java/com/github/lombrozo/testnames/rules/LineHitterRule.java

```diff
@@ -39,7 +39,7 @@
  * the following reasons:
  * "Class 'RuleLineHitter' has bad naming, class ends with '-er' suffix"
  * You can read more about this issue
- * <a href="https://github.com/l3r8yJ/oop-cop/issues/105">here</a>
+ * <a href="https://github.com/l3r8yJ/oop-cop/isslues/105">here</a>
```

Comment:
@h1alexbel Is it typo here? I guess, it should be `issues`

Reply:
@volodya-lombrozo thanks for catch, added `typos.yml`